KeepLayout.podspec

```diff
@@ -9,6 +9,35 @@ Pod::Spec.new do |s|
 
   s.source       = { :git => "https://github.com/Tricertops/KeepLayout.git", :tag => "v1.7.0" }
 
-  s.source_files = 'Sources'
   s.requires_arc = true
+
+  s.module_name = "KeepLayout"
+  s.default_subspec = 'Swift'
+
+  s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.9'
+  s.tvos.deployment_target = '9.0'
+
+  s.subspec 'ObjC' do |op|
+    op.source_files = 'Sources'
+    op.exclude_files = 'Sources/*.{swift}'
+    op.ios.deployment_target = '5.0'
```

Comment:
Yes, I need line 17 to build Swift spec, but I can build ObjC spec with lower deployment target.